src/views/Dashboard/Customer/Home/RequestTest/ServiceDetailDialog.vue

```diff
@@ -6,7 +6,8 @@
           v-icon mdi-close
 
       div.dialog-service__service-image
-        ui-debio-avatar(:src="selectedService.serviceImage" size="120" rounded)
+        v-img(v-if="!selectedService.serviceImage" :src="debioLogo" width="120" height="120" srounded contain)
+        ui-debio-avatar(v-else :src="selectedService.serviceImage" size="120" rounded)
```

Comment:
please add `TODO` here, so in the future, we can improve in a more efficient way. Thank you